db/migrate/20230502113352_create_message.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMessage < ActiveRecord::Migration[7.0]
+  def change
+    create_table :messages do |t|
+      t.text :greeting
+
```

Comment:
- [ ] Great job so far 👏🏻 However the database table for storing different greetings should be created correctly. I can see that you have created the message migration but you should also have created the `message model`. please address this. 👍🏻 

- [ ] Kindly make sure you can correctly Create 5 different greetings using the rails console or seed. 👍🏻